src/services/services.ts

```diff
@@ -2617,6 +2618,14 @@ module ts {
                 entries: activeCompletionSession.entries
             };
 
+            function getScopeNode(initialToken: Node, position: number, sourceFile: SourceFile) {
+                var scope = initialToken;
+                while (scope && (isToken(scope) || !positionBelongsToNode(scope, position, sourceFile))) {
+                    scope = scope.parent;
+                }
+                return scope;

```

Comment:
We actually don't need to since `undefined` is an acceptable value for `getSymbolsInScope` (i.e. there was no given token, so it will default to the file's globals.